src/JITModule.cpp

```diff
@@ -468,8 +472,7 @@ void JITModule::memoization_cache_set_size(int64_t size) const {
 }
 
 bool JITModule::compiled() const {
-    // TODO: Track down all uses and make sure changing this to not include "module != NULL" doesn't break anything.
-  return jit_module.ptr->module != NULL;
+  return jit_module.ptr->execution_engine != NULL;
```

Comment:
Not that I can recall. It seems to be used in some places without causing build failures.